spring-session/src/main/java/org/springframework/session/data/mongo/config/EnableMongoHttpSession.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2002-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.session.data.mongo.config;
```

Comment:
The package should be `org.springframework.session.data.mongo.config.annotation.web.http` to isolate the web based dependencies (there may eventually be an EnableMongoSession annotation for things like Spring Batch which don't have HTTP).